src/cr/cube/dimension.py

```diff
@@ -330,6 +335,10 @@ def __init__(self, dimension_dict, dimension_type, dimension_transforms=None):
         self._dimension_type = dimension_type
         self._unshimmed_dimension_transforms_dict = dimension_transforms or {}
 
+    def __repr__(self) -> str:
+        """str representation of this dimension."""
+        return f"{self.name} ({self.dimension_type}): {self.valid_elements}"
```

Comment:
I would not put the `valid_elements` here in the `__repre__` for 1 reason:
Real cubes have tons of valid elements especially MR and this repr could be not readable.

IMHO this should be only `name` and `dim_type`